src/main/java/org/owasp/esapi/reference/DefaultSecurityConfiguration.java

```diff
@@ -1381,29 +1381,29 @@ public String getStringProp(String propertyName) throws ConfigurationException {
 
     protected boolean shouldPrintProperties() {
         return getESAPIProperty(PRINT_PROPERTIES_WHEN_LOADED, false);
-	}
+    }
 
     protected Properties getESAPIProperties() {
         return properties;
     }
     
     public enum DefaultSearchPath {
-    	
-    	RESOURCE_DIRECTORY("resourceDirectory/"),
-    	SRC_MAIN_RESOURCES("src/main/resources/"),
-    	ROOT("/"),
-    	DOT_ESAPI(".esapi/"),
-    	ESAPI("esapi/"),
-    	RESOURCES("resources/");
-    	
-    	private final String path;
-    	
-    	private DefaultSearchPath(String s){
-    		this.path = s;
-    	}
-    	
-    	public String value(){
-    		return path;
-    	}
+        
+        RESOURCE_DIRECTORY("resourceDirectory/"),
+        SRC_MAIN_RESOURCES("src/main/resources/"),
+        ROOT(""),
```

Comment:
@xeno6696 Can you confirm that this is the correct value?

Reply:
Doesn’t look right.  I would have just restored the value I clobbered in the if block

Reply:
Unfortunately it is the right value. As you can read here: 
http://www.thinkplexx.com/learn/howto/java/system/java-resource-loading-explained-absolute-and-relative-names-difference-between-classloader-and-class-resource-loading

"There are ClassLoader.getResource() and Class.getResource() and they work differently [...] The name should not have a leading “/”."

Please also check the new added test in DefaultSecurityConfigurationTest.

Reply:
@JoergAdler Sorry for the delay.  My concern is that what you did here, isn't what was done in the past:

```
					// try root directory path given by user
					if (in == null){
						in = loaders[i].getResourceAsStream(fileName);
					}
					
```
Which looked erroneous to me, but it IS what we _were_ doing.  I think if we at least re-include that case I'm fine with the rest of the changes.  I'm still not sure why that case was referred to as "root."  I wanted to debug the old 2.1.0.1 release but that would have required me getting an older JDK and I didn't care all that much.